src/acl.c

```diff
@@ -1580,15 +1578,31 @@ sds ACLLoadFromFile(const char *filename) {
             continue;
         }
 
-        /* Try to process the line using the fake user to validate if
-         * the rules are able to apply cleanly. At this stage we also
-         * trim trailing spaces, so that we don't have to handle that
-         * in ACLSetUser(). */
-        ACLSetUser(fakeuser,"reset",-1);
+        /* Get the user, verifying that the default user has only
+         * been updated once from the original value. */
```

Comment:
@madolson Maybe we could avoid this special case by simply creating the default user *after* loading the file, if it was not explicitly created?

Reply:
Ack.